### PR TITLE
Let the qdevice be created by the habootstrap-formula

### PR DESCRIPTION
**Pull request by aleksei-burlakov**
The pacemaker-deploy has previously create the corosync-qdevice
himselt after creating the cluster by the habootstrap-formula.
Now, the habootstrap-formula will create the cluster together with
the qdevice. The qnet node however (the tiebreaker) will still be
created by the pacemaker-deploy.

### CODE REVIEW

**Comment by aleksei-burlakov** [comment]
Need some changes

File: salt/grains.j2 [hunk 1]
```diff
@@ -75,18 +75,14 @@ sbd_disk_device: "{{ sbd.device if common.shared_storage_type == "shared-disk" e
 {% endif %}
 
 
-iscsi_enabled: {{ jsonify(true) if common.shared_storage_type == "iscsi" else jsonify(false) }}
```
Comment (arbulu89) [inline salt/grains.j2:78]: Why have you removed this `iscsi` elements?
I guess we will still need to use iscsi at some cases, they should coexist in my opinion.

File: deploy.py [hunk 2]
```diff
@@ -447,7 +447,11 @@ def provision_execute(name):
     for stage in stages:
         logging.info(f"Running stage")
         results = []
-        with concurrent.futures.ThreadPoolExecutor() as executor:
+        max_executors = len(hosts)
+        # The last stage should be synchronous
```
Comment (arbulu89) [inline deploy.py:451]: We should add the `why` to this sentence to understand in the future the logic about this

File: salt/grains.j2 [hunk 3]
```diff
@@ -75,18 +75,14 @@ sbd_disk_device: "{{ sbd.device if common.shared_storage_type == "shared-disk" e
 {% endif %}
 
 
-iscsi_enabled: {{ jsonify(true) if common.shared_storage_type == "iscsi" else jsonify(false) }}
-
-iscsi_name: "{{  iscsi.name if "iscsi" in env }}"
-
-iscsi_ip: "{{  iscsi.private_ip if "iscsi" in env }}"
-
 qdevice_enabled: {{ jsonify(qdevice.enabled) if "qdevice" in env else jsonify(false) }}
```
Comment (arbulu89) [inline salt/grains.j2:78]: What happens now with this `qdevice_enabled` and the others?
Are they still used?

**Review by arbulu89 — COMMENTED** [review]
Thanks @aleksei-burlakov ,
Some comments from my side

**Comment by MalloZup** [comment]
@arbulu89 this PR was closed see next

**Comment by stefanotorresi** [comment]
Please, link the superseding PRs when closing, otherwise we might lose track of the discussion! ;)

**Comment by aleksei-burlakov** [comment]
This PR was superseded by the #8

**Pull request closed without merging**